tools/print-evaluated-rules.js

```diff
@@ -20,6 +27,8 @@ const { ESLint } = require('eslint');
     global.console.log(JSON.stringify(configTypescriptTypechecked, undefined, 4));
     global.console.log('Evaluated Angular rules:');
     global.console.log(JSON.stringify(configAngular, undefined, 4));
+    global.console.log('Evaluated diverged Angular rules:');
+    global.console.log(JSON.stringify(configAngularExtra, undefined, 4));
 
     global.console.log('-------- Audits:');
     global.console.log(`Evaluated ESLint rules has warn?: ${hasWarn(configEslint)}`);
```

Comment:
Can we add an audit line for these changes? maybe `Has Angular extra rules that are not overwritten?: `

Reply:
Yes.